demonstrations/tutorial_how_to_use_registers.metadata.json

```
@@ -0,0 +1,42 @@
+{
+    "title": "How to use wire registers",
+    "authors": [
+        {
+            "id": "austin_huang"
```

```suggestion
            "username": "austingmhuang"
```

as per @ikurecic's comment in #1167